Portal_Application/Startup.cs

```diff
@@ -15,20 +19,26 @@ public Startup(IConfiguration configuration)
             Configuration = configuration;
         }
 
-        public IConfiguration Configuration { get; }
+        private IConfiguration Configuration { get; }
 
-        // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
-            services.AddControllersWithViews();
-            // In production, the Angular files will be served from this directory
-            services.AddSpaStaticFiles(configuration =>
+            services.AddDbContext<ApplicationContext>(options => options
+                .UseSqlServer(Configuration.GetConnectionString("DefaultConnection")));
+            services.AddSpaStaticFiles(configuration => { configuration.RootPath = "ClientApp/dist"; });
+            services.AddControllers();
+            
+            var config = new MapperConfiguration(mc =>
```

Comment:
That mapper creation looks a little bit mess
Can it be simpler smth like `services.AddAutoMapper(Assembly.GetAssembly(typeof(MappingProfile)));`

Reply:
It doesn't work